include/git2/branch.h

```diff
@@ -34,6 +34,8 @@ GIT_BEGIN_DECL
  *
  * @param out Pointer where to store the underlying reference.
  *
+ * @param repo the repository where to create the branch.
```

Comment:
A little awkward, would rephrase as:

```suggestion
 * @param repo the repository to create the branch in.
```